contentcuration/contentcuration/frontend/shared/leUtils/FormatPresets.js

```diff
@@ -90,7 +90,7 @@ const FormatPresetsMap = new Map([
       thumbnail: false,
       subtitle: false,
       display: true,
-      order: 2,
+      order: 1,
```

Comment:
Seems to me `epub` still has conflicting `order` with PDF documents

Reply:
Order has been updated to match [le-utils presets](https://github.com/learningequality/le-utils/blob/main/le_utils/resources/presetlookup.json). Le-Utils will also be updated in a separate pr with bloom being moved to 3 (initially 1) and document thumbnail moved to 4 (initially 3)

Reply:
https://github.com/learningequality/le-utils